### PR TITLE
removes glass sheets and cable from machine crafting requirement

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 

del: glass sheets and cable coils are no longer required in most machine building
tweak: announcement machine and stacking machine requires different parts

/:cl:

why: glass sheets are just bloat and dont make stuff interesting or engaging
 same for cable coils also the machine frame already uses 5 cable pieces why does a machine need 1 more ??? just dumb defunct design (yes machines were supposed to have shit like proxy sensors, etc source: 2015 comments) that should have been deleted when console screens were removed
also they dont fit in the fucking rped making machine building more painful than already is

### CODE REVIEW

**Comment by Cyberboss** [comment]
You have an argument for cable, but removing glass when most machines _clearly have glass parts_?

**Comment by Shardj** [comment]
You've done way more than you describe in your comment. You've basically halfed the building requirements of tons of stuff. In some you removed stock parts which are needed for upgrades. And why, it seems pretty balanced currently.

**Comment by Tlaltecuhtli** [comment]
where?

**Comment by MMMiracles** [comment]
what about instead of removing the shit because it don't fit in the rped you make it fit in the rped

**Comment by Tlaltecuhtli** [comment]
i ll do that for glass but cable is bad

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -3,42 +3,35 @@
 	build_path = /obj/machinery/sleeper
 	req_components = list(
 		/obj/item/stock_parts/matter_bin = 1,
-		/obj/item/stock_parts/manipulator = 1,
```
Comment (Shardj) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: Removing stock parts isn't part of your description

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 2]
```diff
@@ -3,42 +3,35 @@
...
-		/obj/item/stack/sheet/glass = 2)
+		/obj/item/stock_parts/manipulator = 1)
 
 /obj/item/circuitboard/machine/vr_sleeper
 	name = "VR Sleeper (Machine Board)"
 	build_path = /obj/machinery/vr_sleeper
 	req_components = list(
 		/obj/item/stock_parts/manipulator = 1,
-		/obj/item/stack/cable_coil = 1,
-		/obj/item/stock_parts/scanning_module = 2,
```
Comment (Shardj) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: Removing stock parts isn't part of your description

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 3]
```diff
@@ -3,42 +3,35 @@
...
-		/obj/item/stack/sheet/glass = 1)
+		/obj/item/stock_parts/subspace/amplifier = 1,
+		/obj/item/stock_parts/subspace/crystal = 1)
 
 /obj/item/circuitboard/machine/autolathe
 	name = "Autolathe (Machine Board)"
 	build_path = /obj/machinery/autolathe
 	req_components = list(
 		/obj/item/stock_parts/matter_bin = 3,
-		/obj/item/stock_parts/manipulator = 1,
```
Comment (Shardj) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: Removing stock parts isn't part of your description

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 4]
```diff
@@ -3,42 +3,35 @@
...
-		/obj/item/stack/sheet/glass = 1)
+		/obj/item/stock_parts/manipulator = 1)
 
 /obj/item/circuitboard/machine/clonepod
 	name = "Clone Pod (Machine Board)"
 	build_path = /obj/machinery/clonepod
 	req_components = list(
-		/obj/item/stack/cable_coil = 2,
 		/obj/item/stock_parts/scanning_module = 2,
-		/obj/item/stock_parts/manipulator = 2,
```
Comment (Shardj) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: Removing stock parts isn't part of your description

**Comment by Shardj** [comment]
Also another 20ish places which i wont comment because I think you can see it now

**Comment by 81Denton** [comment]
You can also get free sheets by deconstructing any of the 500 windows and glass tables on each map

**Comment by optimumtact** [comment]
bohoo game too hard, me nerf

**Pull request closed without merging**